pyanaconda/anaconda.py

```diff
@@ -83,7 +83,7 @@ def payload(self):
             payload_type = self._get_dbus_payload_type()
 
             if payload_type == PAYLOAD_TYPE_RPM_OSTREE:
-                if FlatpakPayload.is_available():
+                if FlatpakManager.is_available():
```

Comment:
I am not sure if the `is_available` name still makes sense after the renaming. Maybe it could be `is_supported`?

Reply:
`is_supported` sounds to me like a (abstract) class method that could tell me if I should even instantiate the manager.

What this method tells me is, do the directories the manager "manages" give us some flatpak? (It checks only existence of dir though!) So I think `is_correctly_configured` might work better?

Reply:
Or we can keep the payload in the name and name it `is_payload_available` or `is_source_available`.

Reply:
Thanks, the source variant sounds good to me.

Reply:
Done.